src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -2656,12 +2610,8 @@ private Task ProcessItemAsync(TOutput item)
                         if (_tempSendAsyncTaskList != null && _tempSendAsyncTaskList.Count > 0)
                         {
                             // Consolidate all SendAsync tasks into one
-                            var allSendAsyncTasksConsolidated =
-#if PRENET45
-                            Task.Factory.ContinueWhenAll((Task[])_TempSendAsyncTaskList.ToArray(), s_whenAllAction);
-#else
-                            Task.WhenAll(_tempSendAsyncTaskList);
-#endif
+                            var allSendAsyncTasksConsolidated = Task.WhenAll(_tempSendAsyncTaskList);
```

Comment:
Yes I meant to say "explicitly type" to Task. It wasn't obvious to me what the type was so I had to dig deeper.